Taskfile.yml

```diff
@@ -247,6 +251,7 @@ tasks:
       - "{{.TASKFILE_DIR}}/Taskfile.yml"
       - ".gitmodules"
       - "tools/scripts/deps-download/**/*"
+      - "/etc/os-release"
```

Comment:
I haven't found any evidence but I assume the Boost library is OS dependent. In general, I believe it is good we "rebuild everything" if the OS building platform changes.

Reply:
Submodules should all be source dependencies rather than binaries dependencies, but fair enough.

Reply:
I missed that the task only downloads the dependencies but does not build those. Then it might make more sense to exclude the `/etc/os-release` from the source list.